README.md

```diff
@@ -206,9 +211,8 @@ When commuting to and from the office, please be mindful of the environment. If
 
 ### Taking care of people \(that's you\)
 
-#### PSYA
-
-mondora has partnered with PSYA to offer all employees and their families a free and anonymous counselling service. PSYA has seven trained psychologists who are available on the phone 24/7. You can call them and speak to someone for both work related and personal issues and you don’t have to give your real name. There is also a live chat service available in the daytime through PSYA’s website. Please check out [this presentation](https://docs.google.com/presentation/d/12mUO43tkFL8HwsQEaanNM-rjHYEk-Av5EV_vkvlnpOQ/edit?usp=sharing) for all the info.
+#### Ask me anything
+Openness is one of our core values; we believe that we can be open with our colleagues and share almost anything. Ask me anything is a moment where your colleagues can ask you anything. There is a slack channel #ama where you can post your requests (also anonymously) and you or your mate can answer to all the questions. You can host an AMA or organize it.
```

Comment:
Openness is one of our core values; we believe that we can be open with our colleagues and share almost anything. Ask me anything is a moment in which your colleagues can ask you any question. There is a slack channel #ama in which you can post your questions (also anonymously) and you or your colleagues can choose to answer the questions. You can host an AMA or organize it whenever you like!